hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -294,9 +303,20 @@ checkClose ctx headContext parties initialUtxoHash snapshotNumber closedUtxoHash
        in verifySnapshotSignature parties snapshotNumber closedUtxoHash sig
             && checkHeadOutputDatum ctx expectedOutputDatum
     | otherwise = traceError "negative snapshot number"
+
+  cp = fromMilliSeconds (milliseconds cperiod)
```

Comment:
can cp be zero?

Reply:
It can if you set it to be zero. Another thought - write a test where cp is 0 .

Reply:
0 is member of Natural numbers, but i am not sure we should accept a cp = 0

Reply:
I added a mutation test that exercises zero value for CP